tern/__main__.py

```diff
@@ -36,7 +36,7 @@
 logger.addHandler(log_handler)
 
 
-def main(args):
+def do_main(args):
```

Comment:
I'm actually not familiar with setuptools so I had to do a bit of reading up. Is this change necessary for this PR? Can it exist in a follow-up one?

Reply:
It's necessary for setuptools. entrypoints require a function to call to create the executable stub, so I moved the if __main__ into a function called main. I wonder if another name would've been more convenient/descriptive...

Reply:
`entry_point`?

Reply:
[yep](https://github.com/vmware/tern/pull/190/files/edaff38ab10761ae2b2c3524ea37fdbfe684f477#diff-2eeaed663bd0d25b7e608891384b7298R37), it is of the form [name-of-executable]:[path-to-module.function]

(p.s. sorry I'm in a 60% keyboard and I can't add any syntactic sugar to my comments rn :[ )